features/admin.rules.v1/api/use-get-resource-list.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
```

Comment:
```suggestion
 * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
```

Reply:
Done. But this file was created in 2024 and the feature is moving now. But anyway better update it.